package.json

```diff
@@ -42,10 +42,10 @@
       },
       "brave-core": {
         "dir": "src/brave",
-        "branch": "master",
         "repository": {
           "url": "https://github.com/brave/brave-core.git"
-        }
+        },
```

Comment:
can we eliminate this diff just to avoid other conflicts?

Reply:
👍